docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc

```diff
@@ -0,0 +1,301 @@
+
+You'll be able to login to the PacketFence web administration interface with the credentials created in Step 4.
+
+Services status will help you monitor if everything goes as expected. If not, you'll see which service is in trouble and the log output will help you determine the problem that occurs.
+
+Configuring AeroHIVE controller in PacketFence
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Now that you have a fully functionnal PacketFence installation, you'll need to add the AeroHIVE AP to the PacketFence switches database for correct intergration.
+
+To do so, login to the PacketFence web administration interface if it is not already done. Click on the Configuration tab and select the switches section.
+
+We'll use the clone function to add a new switch (the AeroHive AP) to the PacketFence database. With the mouse pointer, go over the default switch and you'll see a set of icons appearing at the left of it. Click on the second one (a paper sheet with a plus sign).
+
+Confirgure the switch like the following:
+
+image::docs/images/aerohive-pf-config-1.png[scaledwidth="100%",alt="General switch settings"]
+
+image::docs/images/aerohive-pf-config-role.png[scaledwidth="100%",alt="Roles switch settings"]
+
+image::docs/images/aerohive-pf-config-snmp.png[scaledwidth="100%",alt="SNMP switch settings"]
+
+Modify your RADIUS secret.
+
+Click Add switch
+
+The newly added controller should shows up in the list.
+
+PacketFence configuration is done. You may now reboot the demonstation PC (leave the USB key in). Once rebooted, the web browser should open in the PacketFence web administation interface.
+
+Step 3: Configure AeroHIVE
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+This section will discuss about the configuration of your AeroHIVE AP in order to use it with our configured PacketFence environment.
+
+Connect APs and Controller
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+First thing to do is to connect the HiveManager. Like you saw in Step 2, make sure to connect the AP on the TRUNK port configured for it.
+
+Provision the HiveManager
+^^^^^^^^^^^^^^^^^^^^^^^^^
+By default, the HiveManager might be empty, add your AP on it via it's serial number. If necessra update the firmware of the HiveManager and the AP.
+
+NOTE: Please refer to the AeroHIVE Quick Devices Start Guide to add devices: http://docs.aerohive.com/330000/docs/help/english/documentation/AerohiveDevicesQuickStart.pdf
+
+NOTE: Please refer to the following guide to learn more about the HiveManager: http://docs.aerohive.com/330000/docs/help/english/documentation/Aerohive_HiveManagerNG_GettingStartedGuide.pdf
+
+Setup the SSIDs
+^^^^^^^^^^^^^^^
+For this guide, you will show you how to configure one open SSID with RADIUS mac filtering, and one secure SSID using WPA2-Enterprise.
```

Comment:
In this guide, we will configure one open SSI using RADIUS MAC-based filtering and one secured SSID using WPA2 Enterprise.